python/ql/lib/semmle/python/pointsto/PointsTo.qll

```diff
@@ -2125,7 +2125,7 @@ module Conditionals {
 /** INTERNAL: Do not use. */
 predicate declaredAttributeVar(PythonClassObjectInternal cls, string name, EssaVariable var) {
   name = var.getName() and
-  var.getAUse() = cls.getScope().getANormalExit()
+  pragma[only_bind_into](pragma[only_bind_into](var).getAUse()) = cls.getScope().getANormalExit()
```

Comment:
Interesting construction :-)

Reply:
Yeah, it's not the most elegant construction, but a single `pragma` just didn't do the trick.